pynicotine/gtkgui/userbrowse.py

```diff
@@ -294,8 +293,6 @@ def __init__(self, userbrowses, user):
             )
         else:
             self.folder_popup_menu.add_items(
-                ("#" + _("_Download Folder"), self.on_download_folder),
-                ("#" + _("Download Folder _To…"), self.on_download_folder_to),
                 ("#" + _("Download Folder & Subfolders"), self.on_download_folder_recursive),
                 ("#" + _("Download Folder & Subfolders To…"), self.on_download_folder_recursive_to),
```

Comment:
The `D` and `T` keyboard accelerators are gone, perhaps they can be transposed into the remaining menu items instead.

Reply:
Will do once string freeze is over.